modules/cli-manager-updating-plugin.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-updating-plugin_{context}"]
+= Updating CLI plugins with the {cli-manager}
+
+You can update a plugin that was installed for the OpenShift CLI (`oc`) with the {cli-manager}.
```

Comment:
If no action has taken, Krew periodically checks the updates from index and performs this automatically in background. Would be better to mention this.

Reply:
I checked from the Krew's code. This happens when user invokes some operations in Krew (but not clear). Maybe we don't need to mention this. Just leave the doc as is.